samples/CpsExtension/CpsExtension.Vsix/Properties/launchSettings.json

```diff
@@ -1,10 +1,12 @@
 ﻿{
-  "profiles": {
-    "CpsExtension": {
-      "commandName": "Executable",
-      "executablePath": "$(DevenvDir)devenv.exe",
-      "commandLineArgs": "/rootsuffix exp",
-      "environmentVariables": { "CpsExtensionDesignTimeTargetsPath": "$(OutputPath)BuildSystem\\CpsExtension.DesignTime.targets" }
+    "profiles": {
+        "CpsExtension": {
+            "commandName": "Executable",
+            "executablePath": "$(DevenvDir)devenv.exe",
+            "commandLineArgs": "/rootsuffix exp",
+            "environmentVariables": {
```

Comment:
Could include vars that turn on diagnostics. eg:

https://github.com/dotnet/project-system/blob/28a6e1a6fcd3e9985913230ebe296f29a0189388/setup/ProjectSystemSetup/Properties/launchSettings.json#L11-L15